app/src/components/market/market_wizard_creator_container.tsx

```diff
@@ -37,20 +37,31 @@ const MarketWizardCreatorContainer: FC = () => {
           throw new Error('resolution time was not specified')
         }
         const user = await provider.getSigner().getAddress()
-        const { collateral, arbitrator, question, resolution, funding, outcomes, category } = data
+        const {
+          collateral,
+          arbitrator,
+          question,
+          resolution,
+          funding,
+          outcomes,
+          category,
+          loadedQuestion,
+        } = data
         const openingDateMoment = moment(resolution)
 
         const collateralService = new ERC20Service(provider, account, collateral.address)
 
         setMarketCreationStatus(MarketCreationStatus.postingQuestion())
-        const questionId = await realitio.askQuestion(
-          question,
-          outcomes,
-          category,
-          arbitrator.address,
-          openingDateMoment,
-          networkId,
-        )
+        const questionId = !loadedQuestion
```

Comment:
I don't mind this a lot, but _maybe_ it would be better to be more explicit:

```
let questionId: string
if (loadedQuestionId) {
  questionId = loadedQuestionId
} else {
  setMarketCreationStatus(MarketCreationStatus.postingQuestion())
  questionId = await realitio.askQuestion(...)
}
setQuestionId(questionId)
```

(or something like that)

Reply:
Done